src/gui/settings/foundObjects.html

```diff
@@ -165,16 +184,34 @@
      * This gets triggered by a POST message coming into this iframe from the parent window.
      * It receives a set of information for each discovered object, and renders a table displaying the information.
      * This should get called on a (infrequent) loop as long as the iframe is open, so that it doesn't keep stale data.
-     * @param {Object.<string, {name: string, version: string, ip: string, initialized: boolean, frames: Object, isAnchor: boolean, isWorld: boolean}>} msg
+     * @param {Object.<string, {name: string, version: string, ip: string, port: string, initialized: boolean, frames: Object, isAnchor: boolean, isWorld: boolean}>} msg
      */
     realityEditor.gui.settings.getObjectsFunction = function (msg) {
         // don't re-render if nothing has changed
         if (previousFoundObjects && JSON.stringify(previousFoundObjects) === JSON.stringify(msg)) {
             return;
         }
+        
+        var ignoredAnchors = [];
```

Comment:
Can you add a short comment about why we have ignoredAnchors? Is this because the object's ID changes when we upload a new target image? We should eventually fix that on the server but this works in the short-term..